exp_test.go

```diff
@@ -0,0 +1,228 @@
+package iso8583_test
```

Comment:
May I ask you to move the tests into `message_test` and to review if all test cases you may think about are covered by these tests?